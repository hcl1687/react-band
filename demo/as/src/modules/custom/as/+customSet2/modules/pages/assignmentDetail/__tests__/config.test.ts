import configFactory from '../config'

const config = configFactory()

describe('custom/as/pages/assignmentDetail/config', () => {
  it('should render correctly', () => {
    expect(config).toMatchSnapshot()
  })
})
