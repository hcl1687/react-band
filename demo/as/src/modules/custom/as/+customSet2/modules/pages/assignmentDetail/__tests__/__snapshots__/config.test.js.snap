// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`custom/as/pages/assignmentDetail/config should render correctly 1`] = `
Object {
  "decorators": Array [
    "@i18n",
    "@theme",
    "@reduxStore",
  ],
  "decoratorsConfig": Object {
    "@reduxStore": Object {
      "assignmentStore": Object {
        "actions": Array [
          "getAssignment",
          "editAssignment",
        ],
        "state": Array [
          "assignment",
        ],
      },
      "breadcrumbStore": Object {
        "actions": Array [
          "setBreadcrumb",
        ],
      },
    },
  },
  "name": "assignmentDetail",
}
`;
